test/integration/test-example-validation.js

```diff
@@ -55,6 +55,7 @@ describe.configure().retryOnSaucelabs().run('example', function() {
     'facebook.amp.html',
     'gfycat.amp.html',
     'instagram.amp.html',
+    'ooyala.amp.html',
```

Comment:
this file doesnt exist right? think you might need to rebase from master, i tried to figure out how to fix your test issues and there were some conflicts for this file. let me know when you've rebased then we can try and figure out how to fix the issues.

think you can just reset test-example-validation.js
and then resolve gulpfile.js as well, (might need to reset and then just re-add the amp-ooyala entry)

we're close!